src/utils/url-test.js

```diff
@@ -258,3 +260,77 @@ describe('addQueryParam', () => {
     expect(addQueryParam('file?d=e#f', 'a', 'b')).to.equal('file?d=e&a=b#f');
   });
 });
+
+describe('serializeProtoMessageForUrl', () => {
+  it('should serialize message', () => {
+    const inputArray = [
+      null,
```

Comment:
Instead of `null` could you add something that looks like a `label` for both tests?